dbt_subprojects/tokens/models/transfers_and_balances/abstract/tokens_abstract_net_transfers_daily_asset.sql

```diff
@@ -0,0 +1,19 @@
+{% set blockchain = 'abstract' %}
+
+{{ config(
+        schema = 'tokens_' + blockchain
+        , alias = 'net_transfers_daily_asset'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date', 'contract_address']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+
+{{ evm_net_transfers_daily_asset(
+        blockchain=blockchain,
+        native_contract_address=var('ETH_ERC20_ADDRESS')
```

Comment:
we shouldn't assume all EVM chains use this variable. plz double check each new addition against their `tokens_<blockchain>_base_transfers.sql` equivalent.

for instance, abstract has a different value
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/tokens/models/transfers_and_balances/abstract/tokens_abstract_base_transfers.sql#L18

Reply:
quick check against all chains in PR would be good 🙏

Reply:
reminder here on this one @beve0x